src/btest.cr

```diff
@@ -329,73 +277,56 @@ class Case
 
     if validation_errors.size > 0
       return Result.new(runner, self, false,
-                        validation_errors +
-                        "\nstdout: #{stdout}\nstderr: #{stderr}")
+        validation_errors + "\nstdout: #{stdout}\nstderr: #{stderr}")
     end
 
     Result.new(runner, self, true, "")
   end
 end
 
-
 # This is the result of a test case run with a particular runner.
 class Result
-  def initialize(runner : Runner, testCase : Case, pass : Bool,
-                 message : String)
-    @runner = runner
-    @testCase = testCase
-    @pass = pass
-    @message = message
-  end
+  getter testCase, pass, message
 
-  def testCase
-    @testCase
-  end
-
-  def pass
-    @pass
-  end
-
-  def message
-    @message
+  def initialize(@runner : Runner, @testCase : Case, @pass : Bool,
+                 @message : String)
   end
 
   def render
     name = @testCase.name.as(String)
 
     uncolored_len = @testCase.suite.name.size + 1 + @runner.name.size + 1 + \
-                    name.size
+      name.size
 
-    # Chop the output if the name is too long
-    chop_delta = uncolored_len - (TERMINAL_WIDTH - 7)
-    if chop_delta > 0
-      name = name[0..name.size - chop_delta]
-      uncolored_len -= chop_delta - 1
-    end
+  # Chop the output if the name is too long
+  chop_delta = uncolored_len - (TERMINAL_WIDTH - 7)
+  if chop_delta > 0
+    name = name[0..name.size - chop_delta]
+    uncolored_len -= chop_delta - 1
+  end
 
-    suite = "#{@testCase.suite.name}".colorize(:white)
-    runner = "#{@runner.name}".colorize(:dark_gray)
-    ret = "#{runner} #{suite} #{name}"
+  suite = "#{@testCase.suite.name}".colorize(:white)
```

Comment:
you don't need the string interpolation here: `@testCase.suite.name.colorize(:white)`

(ditto bellow)